### PR TITLE
windows py3 abi3 wheels + new workflow invocation

### PR DESCRIPTION
**Pull request by reaperhulk**

### CODE REVIEW

File: .github/workflows/wheel-builder.yml [hunk 1]
```diff
@@ -93,12 +95,14 @@ jobs:
           - {VERSION: "3.5", SODIUM_MSVC_VERSION: "v140"}
           - {VERSION: "3.6", SODIUM_MSVC_VERSION: "v140"}
           - {VERSION: "3.7", SODIUM_MSVC_VERSION: "v140"}
-          - {VERSION: "3.8",  SODIUM_MSVC_VERSION: "v140"}
+          - {VERSION: "3.8", SODIUM_MSVC_VERSION: "v140"}
+          - {VERSION: "3.8", SODIUM_MSVC_VERSION: "v140", "USE_ABI3": "true", "ABI_VERSION": "cp36"}
     name: "Python ${{ matrix.PYTHON.VERSION }} on Windows ${{ matrix.WINDOWS.ARCH }}"
+    name: "${{ matrix.PYTHON.VERSION }} ${{ matrix.WINDOWS.ARCH }} ${{ matrix.PYTHON.AB      I_VERSION }}"
```
Comment (alex) [inline .github/workflows/wheel-builder.yml:101]: cool story

File: .github/workflows/wheel-builder.yml [hunk 2]
```diff
@@ -123,25 +127,37 @@ jobs:
...
+        env:
+          PYNACL_SODIUM_LIBRARY_NAME: sodium
+          PYNACL_SODIUM_STATIC: 1
+          SODIUM_INSTALL: system
+        if: matrix.PYTHON.VERSION != '2.7' && matrix.PYTHON.USE_ABI3 != 'true'
+      - name: Build the wheel (py3 abi3)
         run: |
           call "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\Common7\Tools\VsDevCmd.bat" -no_logo -arch=${{ matrix.WINDOWS.VS_ARCH }}
-          pip wheel pynacl==${{ github.event.client_payload.BUILD_VERSION }} --wheel-dir=wheelhouse --no-binary pynacl
+          cd PyNaCl* && python setup.py bdist_wheel  --py-limited-api=${{ matrix.PYTHON.ABI_VERSION }} && mv dist/PyNaCl*.whl ../wheelhouse
```
Comment (alex) [inline .github/workflows/wheel-builder.yml:154]: is this valid cmd syntax?

File: .github/workflows/wheel-builder.yml (hunk 1 above)
Comment (reaperhulk) [reply .github/workflows/wheel-builder.yml:101]: Look you know I can't paste.

File: .github/workflows/wheel-builder.yml (hunk 2 above)
Comment (reaperhulk) [reply .github/workflows/wheel-builder.yml:154]: Yeah we do this in cryptography without issue.

**Comment by alex** [comment]
Can you trigger a build here, so we can see the results?

**Review by alex — APPROVED** [review]

**Pull request merged**